src/BlazorWebView/src/SharedSource/ExternalLinkNavigationEventArgs.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Microsoft.AspNetCore.Components.WebView
+{
+	/// <summary>
+	/// Used to provide information about a link (<![CDATA[<a>]]>) clicked within a Blazor WebView.
+	/// 
+	/// Anchor tags with target="_blank" will always open in the default
+	/// browser and the ExternalNavigationStarting event won't be called.
```

Comment:
Can't cref the ExternalNavigationStarting event because this class is in the `SharedSource` and is used with `ExternalNavigationStarting` in `Maui`, `WinForms` & `WPF`.